gui/RemoveItem.java

```diff
@@ -216,33 +243,42 @@ public void setup() {
 		// The students on the Roster List for this class
 		ArrayList<ListEntry> rlist = new ArrayList<ListEntry>();
 
-		for (String s : AdminUI.roster) {
+		for (String s : MentorUI.roster) {
```

Comment:
Oh this was my bad. I copied over what i did in the mentor ui so that stuck I can change it quickly